### ISSUE TITLE
Maintenance and overall thoughts

### ISSUE THREAD

**Issue by valdestron** [node-ts/bus#98]
Greetings. First I would like to say big thanks to the owners of this project, it helped a lot for my use cases and I am really happy with performance and overall design. I build some of the tools myself around this package family for my graphql microservice applications and everything works quite well.

Although everything is more less fine, I am a bit concerned if this project is still maintained? I see lots of pull requests hanging, also various dependency warnings for some of the packages. Also I am missing logger package related source code, this could also be available for the community.

Any ideas for further development and support for this project? What about pullrequests, I would like to contribute to this project also, but curious if its not dead and pullrequests would be accepted ?

**Comment by adenhertog** [comment]
Hi @valdestron 

Thanks for getting in touch. I haven't been as active on this project since inception but I do still do bug fixes as they pop up and recently I've been working to remove inversify (https://github.com/node-ts/bus/tree/remove-inversify) as a hard dependency.

The logger code is available at https://github.com/node-ts/logger - but even this is something I'm planning to remove as a hard dependency and move to a more simplistic implementation.

This codebase is used in production and so it'll continue to be maintained as such. I appreciate any pull requests and will happily help get them merged into master - feel free to reach out if you want to discuss anything before starting.

Cheers,
Andrew

**Comment by valdestron** [comment]
Thanks for clarification. I am glad that this project will be maintained in the future and pull requests will be reviewed and accepted. I think this project have a potential, there is not that much in nodejs world that really helps in building custom orchestrators. Closing this issue.
